crates/core/src/config.rs

```diff
@@ -45,6 +45,7 @@ pub struct UserAuth {
     pub access_token: String,
     pub refresh_token: String,
     pub user_address: String,
+    pub expires_at: String,
```

Comment:
expires_at should be a Datetime and while you are at it, we should also change user_address to type Address from alloy. It was a mistake on my part to store it as a String.

All of these types have `serde` implementations so they can be deserialized from a file into the data structure and also serialized from the data structure into a file (in our case a toml file).